care/emr/models/allergy_intolerance.py

```diff
@@ -18,3 +18,4 @@ class AllergyIntolerance(EMRBaseModel):
     copied_from = models.BigIntegerField(
         default=None, null=True, blank=True
     )  # If True, the record is a copy maintained of the given ID
+    type = models.CharField(max_length=100, default="allergy")
```

Comment:
Lets call it allergy_intolerance_type

Reply:
we would also need migration for existing records

Reply:
We added a new attribute right ? it would take the default value.